code/modules/dwarven/dwarven_items.dm

```diff
@@ -0,0 +1,170 @@
+	desc = "A very sharp axe blade made of greatest dwarven metal."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "greyscale_dwarven_axe"
+	item_state = "greyscale_dwarven_axe"
+	lefthand_file = 'icons/mob/inhands/weapons/axes_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/axes_righthand.dmi'
+	flags_1 = CONDUCT_1
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR
+	force = 15
+	throwforce = 16
+
+/obj/item/hatchet/dwarven/axe/Initialize()
+	. = ..()
+	AddComponent(/datum/component/butchering, 60, 80)
+
+/obj/item/hatchet/dwarven/javelin
+	name = "dwarven javelin"
+	desc = "A very sharp javelin made of greatest dwarven metal."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "greyscale_dwarven_javelin"
+	item_state = "greyscale_dwarven_javelin"
+	lefthand_file = 'icons/mob/inhands/weapons/polearms_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/polearms_righthand.dmi'
+	attack_verb = list("attacked", "poked", "jabbed", "torn", "gored")
+	flags_1 = CONDUCT_1
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR
+	force = 7
+	throwforce = 18
+	throw_speed = 4
+	throw_range = 5
+	embedding = list("embedded_pain_multiplier" = 4, "embed_chance" = 80, "embedded_fall_chance" = 20)
+
+/obj/item/dwarven/rune_stone
+	name = "rune stone"
+	desc = "Dwarven magical artifact, looks fragile."
+	icon = 'icons/obj/dwarven.dmi'
+	icon_state = "runestone"
+	var/overlay_state
+	var/mutable_appearance/overlay
+
+/obj/item/dwarven/rune_stone/Initialize()
+	. = ..()
+	if(overlay_state != null)
+		overlay = mutable_appearance(icon, overlay_state)
+		overlay.appearance_flags = RESET_COLOR
+		add_overlay(overlay)
+
+/obj/item/dwarven/rune_stone/proc/apply(atom/target, mob/user)
+
+	return
+
+/obj/item/dwarven/rune_stone/afterattack(atom/target, mob/user, proximity_flag)
+	. = ..()
+	if(overlay_state == null)
+		user.visible_message("<span class='warning'>The runestone is out of charge!</span>")
+		return
+	if(proximity_flag)
+		apply(target, user)
+		overlay_state = null
+		cut_overlays()
+
+/obj/item/dwarven/rune_stone/attack_self(mob/user)
+	if(overlay_state == null)
+		user.visible_message("<span class='warning'>The runestone is out of charge!</span>")
+		return
+	if(user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
+		apply(user, user)
+		overlay_state = null
+		cut_overlays()
+
+/obj/item/dwarven/rune_stone/blitz
+	name = "blitz runestone"
+	overlay_state = "blitz_rune"
+	var/wall_type = /obj/effect/forcefield
+
+/obj/item/dwarven/rune_stone/blitz/apply(atom/target, mob/user)
+	new wall_type(get_turf(user),user)
+	if(user.dir == SOUTH || user.dir == NORTH)
+		new wall_type(get_step(user, EAST),user)
+		new wall_type(get_step(user, WEST),user)
+	else
+		new wall_type(get_step(user, NORTH),user)
+		new wall_type(get_step(user, SOUTH),user)
+	..()
+
+
+/obj/item/dwarven/rune_stone/earth
+	name = "earth runestone"
+	overlay_state = "earth_rune"
+
+/obj/item/dwarven/rune_stone/earth/apply(atom/target, mob/user)
```

Comment:
you can port hippie's thanos shockwave for this
or the repulse spell

Reply:
Or maybe a forcewall-like effect, but opaque?

Reply:
forcewall is already in blitz rune iirc